docs/app/Examples/modules/Dropdown/Usage/index.js

```diff
@@ -119,6 +119,11 @@ const DropdownUsageExamples = () => (
       description='A dropdown can open its menu upward.'
       examplePath='modules/Dropdown/Usage/DropdownExampleUpwardSelection'
     />
+    <ComponentExample
+      title='Wrap Selection'
+      description='A dropdown can wrap its selection to disable "infinite" selection'
```

Comment:
Should this say `enable` rather than `disable`?  Also, what do you think about adding the word "scrolling" into here?  Something "enable infinite scrolling the selection".  Or, "A dropdown selection can wrap to the start when it reaches the end and vice versa"?  Could just be me but I wasn't quite sure what this was described on the first read.

Thoughts?